src/components/Dashboard/TodoList.vue

```diff
@@ -124,47 +124,47 @@
               </div>
             </div>
 
-            <div class="list-item__actions" v-if="!isCorrection(item)">
+            <div class="list-item__actions" v-if="!isCorrection(task)">
               <div style="width:100%">
                 <p class="date-subtitle"
-                  v-if="entityFilter(EntityTypes.BCOMP) && isConfirmEnabled(item.type, item.status)"
-                >Due {{ item.nextArDate }}</p>
+                  v-if="entityFilter(EntityTypes.BCOMP) && isConfirmEnabled(task.type, task.status)"
+                >Due {{ task.nextArDate }}</p>
 
                 <!-- pre-empt any buttons below -->
-                <template v-if="inProcessFiling !== undefined && inProcessFiling === item.id">
+                <template v-if="inProcessFiling !== undefined && inProcessFiling === task.id">
                   <v-btn text loading disabled />
                 </template>
 
-                <template v-else-if="isDraft(item)">
+                <template v-else-if="isDraft(task)">
                   <v-btn class="btn-draft-resume"
                     color="primary"
-                    :disabled="!item.enabled"
-                    @click.native.stop="doResumeFiling(item)"
+                    :disabled="!task.enabled"
+                    @click.native.stop="doResumeFiling(task)"
                   >
                     <span>Resume</span>
                   </v-btn>
                   <!-- more DRAFT actions menu -->
                   <v-menu offset-y left>
```

Comment:
No split button for correction task (until later), per new uxpin linked to #2615.

https://preview.uxpin.com/cefa7724eb7ad8538cb92f016364066b202fbec2#/pages/124203930/simulate/no-panels

Reply:
Thanks, will keep in mind when implementing draft work!